cotix/_bodies.py

```diff
@@ -15,11 +15,13 @@ class AbstractBody(eqx.Module, strict=True):
     mass: AbstractVar[Float[Array, ""]]
     inertia: AbstractVar[Float[Array, ""]]
 
-    position: AbstractVar[Float[Array, "2"]]
+    position: AbstractVar[
+        Float[Array, "2"]
+    ]  # having position here and in shape is inconvenient
     velocity: AbstractVar[Float[Array, "2"]]
 
     angle: AbstractVar[Float[Array, ""]]
-    angular_velocity: AbstractVar[Float[Array, "2"]]
+    angular_velocity: AbstractVar[Float[Array, "2"]]  # shouldnt this be a scalar?
```

Comment:
Just finished ramming jaxtyping through the codebase, and this one one of two mistakes :)